docs/network.md

```diff
@@ -0,0 +1,16 @@
+# Network ports
+
+Percona Platform requires configuration of network ports to accept monitoring data from PMM. 
```

Comment:
It is not really for monitoring. It works in a way that PMM should communicate to platform services for some reason like getting advisor checks downloaded from our check service. So to make this happen PMM team needs to whitelist the specified addresses in their Settings in order for PMM to recognize this communication as a safe one.
"To receive the most value out of Percona Platform, please ensure your corporate security teams whitelist the following IP addresses"